README.md

```diff
@@ -0,0 +1,72 @@
+
+# Sketch to SwiftUI
+
+> Create a StylableSwiftUI stylist implementation from a Sketch file. Sketch -> SwiftUI
+
+## Installation
+
+- Clone this repository
+- Run `npm i`
+
+### Clone
+
+- Clone this repo to your local machine using `https://github.com/design-ops/sketch-to-swiftUI`
+
+### Setup
+
+> Install dependencies first
+
+```shell
+$ brew install fonttools
+```
+
```

Comment:
Do you want to add a note that this is optional, and only required if the fonts found happen to be in a font collection?

Reply:
👍